scene/gui/rich_text_label.cpp

```diff
@@ -4157,6 +4157,18 @@ int RichTextLabel::get_content_height() const {
 	return total_height;
 }
 
+int RichTextLabel::get_content_width() const {
+	int total_width = 0;
+	if (main->lines.size()) {
```

Comment:
This `if` is unnecessary, loop condition already covers `size() == 0` case.

Reply:
Done.